bedwars-api/src/main/java/com/tomkeuper/bedwars/api/language/Messages.java

```diff
@@ -152,6 +152,7 @@ public class Messages {
 
     public static String ARENA_MAX_BUILD_LIMIT_REACHED = "build-max-height-limit-reached";
     public static String ARENA_MIN_BUILD_LIMIT_REACHED = "build-min-height-limit-reached";
+    public static String ARENA_ANNOUNCEMENT = "arena-announcement";
```

Comment:
if you can change `ARENA_ANNOUNCEMENT` and `arena-announcement` to `IN_GAME_ANNOUNCEMENT` and `in-game-announcement` it's perfect.I noticed that only now.

Reply:
maybe ARENA_IN_GAME_ANNOUNCEMENT is better as its an ARENA setting

Reply:
also